### PR TITLE
feat: add husky, lint-staged and .github/workflows/ci.yml file

### PR DESCRIPTION
**Pull request by iryna-mertsalova**
## Links
https://trello.com/c/7ym6cOyf)](https://trello.com/c/7ym6cOyf

## Problem
- Command ` npm run lint ` doesn't work successfuly.
- Missing pre-commit checks

## Solution
- Fixing files, changing values of fileds
- Installed and configured Husky and Lint-Staged
- Set Up GitHub Actions for CI/CD Workflow

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run lint
```
Comment (n-IhorChaikovskyi) [inline .github/workflows/ci.yml:25]: Missing script command --> check fails 
Screenshot 2024-10-31 at 16 25 51

Comment (n-IhorChaikovskyi) [reply .github/workflows/ci.yml:25]: Add a lint command in package.json

Comment (iryna-mertsalova) [reply .github/workflows/ci.yml:25]: Done.

Comment (IhorChai) [reply .github/workflows/ci.yml:25]: Still seeing errors 
Screenshot 2024-10-31 at 16 44 17
Screenshot 2024-10-31 at 16 43 33

File: package.json [hunk 2]
```diff
@@ -6,7 +6,12 @@
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test",
+    "prepare": "husky",
+    "lint": "eslint . && stylelint src/styles/**/*.scss",
```
Comment (IhorChai) [inline package.json:11]: It's better to set a general path for all scss files like this \"src/**/*.{css,scss}\"

You also can run the npm scripts you've created: "lint":  "npm run eslint && npm run stylelint"

Comment (iryna-mertsalova) [reply package.json]: Done

**Review by IhorChai — APPROVED** [review]

**Pull request merged**